### PR TITLE
charts-structure-change

### PR DESCRIPTION
**Pull request by ssyno**
This PR:

- adds extra `helm chart` for the `ciliumNetworkPolicies`
- changes the previous `netpols` `helm chart` to be used only for `networkPolicies` 

### Checklist

- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

File: helm/cert-manager/Chart.yaml [hunk 1]
```diff
@@ -10,10 +10,14 @@ sources:
...
+  - name: Bigmac
 dependencies:
   - name: cert-manager-giantswarm-clusterissuer
     version: 2.0.0
     alias: giantSwarmClusterIssuer
   - name: cert-manager-giantswarm-netpol
-    version: 0.1.0
+    version: 0.2.0
+  - name: cert-manager-giantswarm-ciliumnetworkpolicies
+    version: 0.2.0
```
Comment (kopiczko) [inline helm/cert-manager/Chart.yaml:22]: Do we have to bump those versions?

File: helm/cert-manager/ct.yaml
Comment (kopiczko) [inline helm/cert-manager/ct.yaml:1]: Is that needed?

**Review by kopiczko — APPROVED** [review]

File: helm/cert-manager/Chart.yaml (hunk 1 above)
Comment (ssyno) [reply helm/cert-manager/Chart.yaml:22]: I don't think if this is essential I was just trying to pass linting, will revert on next commit

File: helm/cert-manager/ct.yaml
Comment (ssyno) [reply helm/cert-manager/ct.yaml:1]: I will remove on next commit, I used it for the lint fix

**Pull request merged**